relay-general/src/store/normalize/contexts.rs

```diff
@@ -204,7 +204,17 @@ pub fn normalize_context(context: &mut Context) {
         Context::Runtime(runtime) => normalize_runtime_context(runtime),
         Context::Os(os) => normalize_os_context(os),
         Context::Response(response) => normalize_response(response),
-        _ => (),
+        Context::Device(device) => {
+            if let Some(product_name) = device
+                .as_ref()
+                .model
+                .value()
+                .and_then(|model| ANDROID_MODEL_NAMES.get(model.as_str()))
+            {
+                device.name.set_value(Some(product_name.to_string()))
```

Comment:
I want to highlight that from the earliest commits I changed it from setting the value to "model" to setting it to "name".  

on struct `devicecontext` the namefield just has the comment `Name of the device` which seems a bit unclear but the product name of it sounds like a good fit. It's also not used at all. If you go to sentry discover and filter events by those that have a non-empty device.name field, you get zero results: 

https://sentry.sentry.io/discover/homepage/?field=title&field=event.type&field=project&field=user.display&field=timestamp&field=replayId&name=All+Events&query=%21device.name%3A%22%22&sort=-timestamp&statsPeriod=14d&yAxis=count%28%29

this way we also get  to keep the original model name which could be useful. Let me know if there's any objections to this.